tools/main-transpile-wrapper.js

```diff
@@ -0,0 +1,15 @@
+// This file exists because it is the file in the tool that is not automatically
+// transpiled by Babel
+
+require('babel-core/register')(require('./babel-config.js')); // #RemoveInProd this line is removed in isopack.js
```

Comment:
I don't have a strong opinion about either of those questions.  I trust that you two can figure out the best way to solve the problems, I'm just pointing out that they exist (and/or, potentially will start surprisingly existing in the future when minor things change).